jupyterhub/templates/scheduling/user-scheduler/deployment.yaml

```diff
@@ -15,7 +15,6 @@ spec:
       labels:
         {{- /* Changes here will cause the Deployment to restart the pods. */}}
         {{- include "jupyterhub.matchLabels" . | nindent 8 }}
-        hub.jupyter.org/pod-kind: "core"
```

Comment:
The scheduler is a core pod, no?

Reply:
Yes, but as these labels never were utilized, I removed them